conduit/data/config.go

```diff
@@ -56,6 +56,11 @@ type Telemetry struct {
 	Password string `yaml:"password"`
 }
 
+// API defines parameters for the Conduit API server.
+type API struct {
+	Address string `yaml:"addr"`
```

Comment:
If all we really need or can handle is the port, maybe just change `Address` &rarr; `Port` ?

Reply:
I had it that way originally, but the metrics were defined as a string, and so are the addresses in Indexer/Algod. As a string you can do things like use ipv6 addresses.